docs/architecture/containerized-lifecycle/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -15,74 +16,72 @@ With the latest versions of Docker for Windows, it's easier than ever to develop
 > [!TIP]
 > To learn more about installing Docker for Windows, go to (<https://docs.docker.com/docker-for-windows/>).
 
-## Docker support in Visual Studio 2017
+## Docker support in Visual Studio
 
-There are two levels of Docker support you can add to a project. In ASP.NET Core projects, you can just add a `Dockerfile` file to the project by enabling Docker support. The next level is container orchestration support, which adds a `Dockerfile` to the project (if it doesn't already exist) and a `docker-compose.yml` file at the solution level. Container orchestration support, via Docker Compose, is added by default in Visual Studio 2017 versions 15.0 to 15.7. Container orchestration support is an opt-in feature in Visual Studio 2017 versions 15.8 or later. Version 15.8 an later support Docker Compose and Service Fabric.
+There are two levels of Docker support you can add to a project. In ASP.NET Core projects, you can just add a `Dockerfile` file to the project by enabling Docker support. The next level is container orchestration support, which adds a `Dockerfile` to the project (if it doesn't already exist) and a `docker-compose.yml` file at the solution level. Container orchestration support, via Docker Compose, is added by default in Visual Studio 2017 versions 15.0 to 15.7. Container orchestration support is an opt-in feature in Visual Studio 2017 versions 15.8 or later. Visual Studio 2019 and later also supports **Kubernetes/Helm**.
```

Comment:
Can you explain in VS 2019, what is the kind of support we get instead of explaining the history of support?